README.md

```diff
@@ -1,217 +1,212 @@
-
     repositories {
-        mavenCentral()
+        // ...
+        jcenter()
     }
 
     dependencies {
-    	// ...
+        // ...
         classpath 'com.github.triplet.gradle:play-publisher:1.2.1'
     }
 }
 ```
 
-[![Latest Version](https://maven-badges.herokuapp.com/maven-central/com.github.triplet.gradle/play-publisher/badge.svg)](https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.github.triplet.gradle%22%20AND%20a%3A%22play-publisher%22)
-
-Apply it to your app-level build.gradle file as following below
+Then apply it to each individual `com.android.application` module where you want to use this plugin.
+For example, `app/build.gradle(.kts)` is a common app module:
 
 ```groovy
-apply plugin:'com.android.application'
+apply plugin: 'com.android.application'
 apply plugin: 'com.github.triplet.play'
 ```
 
-The plugin creates the following tasks for you:
+### Sample tasks
+
+Here are some example tasks the plugin might create:
 
 * `publishApkRelease` - Uploads the APK and the summary of recent changes.
 * `publishListingRelease` - Uploads the descriptions and images for the Play Store listing.
 * `publishRelease` - Uploads everything.
 * `bootstrapReleasePlayResources` - Fetches all existing data from the Play Store to bootstrap the required files and folders.
 
-Make sure to set a valid `signingConfig` for the release build type. Otherwise, there won't be a publishable APK and the above tasks won't be available.
-
-In case you are using product flavors you will get one of the above tasks for every flavor. E.g. `publishApkPaidRelease` or `publishListingPaidRelease`.
+Should you choose to use product flavors, there will be an apporpriately named task for each flavor.
+E.g. `publishApkPaidRelease` or `publishListingPaidRelease`.
 
-## Authentication
+## Authenticating the plugin
 
-Similar to Android's `signingConfigs` you need to setup so called `playAccountConfigs` to authorize your requests. Drop in your service account email address and the p12 key file you generated in the API Console here.
+After you've gone though the [Service Account setup](#service-account), you should have a JSON file
+with your private key. Add a `play` block alongside your `android` one with the JSON file's
+location:
 
 ```groovy
-android {
-    playAccountConfigs {
-        defaultAccountConfig {
-            serviceAccountEmail = 'your-service-account-email'
-            pk12File = file('key.p12')
-        }
-    }
-    
-    defaultConfig {
-        // ...
-        
-        playAccountConfig = playAccountConfigs.defaultAccountConfig
-    }
-}
-```
-
-Alternatively you can use a JSON file that can be generated in the API Console.
-
-```groovy
-android {
-    playAccountConfigs {
-        defaultAccountConfig {
-            jsonFile = file('keys.json')
-        }
-    }
-    
-    defaultConfig {
-        // ...
-        
-        playAccountConfig = playAccountConfigs.defaultAccountConfig
-    }
+play {
+    jsonFile = file('your-key.json')
```

Comment:
Should this be the preferred way of authentication for single flavor projects? I remember that we actually wanted to deprecate that solution in favor of `playAccountConfigs`

Reply:
Also, we should keep a reference to pk12 files somewhere (maybe in the advanced section) as long as Google allows to authenticate that way. We probably have to keep that mechanism forever to not force users to recreate service accounts